packages/aws-client/src/test-utils.ts

```diff
@@ -92,38 +98,48 @@ function localStackAwsClient(Service, options) {
  *
  * @param {Function} Service - an AWS service object constructor function
  * @param {Object} options - options to pass to the service object constructor function
- * @returns {Object} - an AWS service object
+ * @returns {Object} an AWS service object
+ *
+ * @private
  */
-function testAwsClient(Service, options) {
-  if (localstackSupportedService(Service)) {
+export function testAwsClient<T extends AWS.Service | AWS.DynamoDB.DocumentClient>(
+  Service: new (params: object) => T,
+  options: object
+): T {
+  // @ts-ignore
+  const serviceIdentifier = Service.serviceIdentifier;
+  if (localstackSupportedService(serviceIdentifier)) {
     return localStackAwsClient(Service, options);
   }
 
-  return {};
+  return <T>{};
 }
-exports.testAwsClient = testAwsClient;
 
-const throwThrottlingException = () => {
-  const throttlingException = new Error('ThrottlingException');
-  throttlingException.code = 'ThrottlingException';
+class ThrottlingException extends Error {
```

Comment:
Is there any reason not to move this to the `errors` package?

Reply:
Moved.